docs/content/index.md

```diff
@@ -2,60 +2,153 @@
+
+`sampleData_2011-01-01T02:00:00:00Z_2011-01-01T03:00:00:00Z_v1_2`
+
+All 3 segments must be loaded before a query for the interval `2011-01-01T02:00:00:00Z_2011-01-01T03:00:00:00Z` completes.
+
+## Loading the Data
+
+Druid currently has a couple of different ways of getting data into it. In the near future, we'd like to address this complexity and unify the various data ingestion methods.
+For now, Druid has a "real-time" method of delivering current time data, and a batch loading mechanism to load static data. Both the real-time and batch loading
+methods deal with creating immutable segments. In the case of real-time ingestion, mutable segments, which are also queryable, are created and periodically finalized into an
+immutable form. In the case of batch ingestion, immutable segments are created directly.
+
+Real-time ingestion in Druid is best effort. Much like other popular data infrastructure open source projects that deal with streams of data,
+exactly once semantics are not guaranteed with real-time ingestion in Druid. Like the other projects, we have it on our roadmap to move towards a streaming only world.
+One common approach to operating Druid is to have a real-time pipeline for recent insights, and a batch pipeline for the accurate copy of the data.
+
+## The Druid Cluster
+
+A Druid Cluster is composed of several different types of nodes. Each node is designed to do a small set of things very well.
+
+### Historical Nodes
+
+Historical nodes commonly form the backbone of a Druid cluster. Historical nodes download immutable segments locally and serve queries over those segments.
+The nodes have a shared nothing architecture and know how to load segments, drop segments, and serve queries on segments.
+
+### Broker Nodes
+
+Broker nodes are what clients and applications query to get data from Druid. Broker nodes are responsible for scattering queries and gathering and merging results.
+Broker nodes know what segments live where.
+
+### Coordinator Nodes
+
+Coordinator nodes manage segments on historical nodes in a cluster. Coordinator nodes tell historical nodes to load new segments, drop old segments, and move segments to load balance.
+
+### Real-time Processing Nodes
+
+Real-time processing in Druid can currently be done using standalone realtime nodes or using the indexing service. The real-time logic is common between these two services.
+Real-time processing involves ingesting data, indexing the data (creating segments), and handing segments off to historical nodes. Data is queryable as soon as it is
+ ingested by the realtime processing logic. The hand-off process is also lossless; data remains queryable throughout the entire process.
+
+### External Dependencies
+
+Druid has a couple of external dependencies for cluster operations.
+
+* **Distributed Coordination** Druid relies on a distributed coordination service for intra-cluster communication. The only supported distributed coordination service right now is Zookeeper.
```

Comment:
Just say ZooKeeper. There's no need to have a layer of abstraction in the docs if we don't have it in the code ;)